### PR TITLE
Update sandman_es.ts

### PR DESCRIPTION
**Pull request by sebadamus**
Some extra text corrections!

### CODE REVIEW

File: SandboxiePlus/SandMan/sandman_es.ts [hunk 1]
```diff
@@ -3181,13 +3181,13 @@ NO seleccionará: %2</translation>
...
+        <source>Note: MSI Installer Exemptions should not be required, but if you encounter issues installing a MSI package which you trust, this option may help the installation complete successfully. You can also try disabling drop admin rights.</source>
+        <translation>Nota: Excepciones para instaladores MSI no deben ser requeridas, pero si encuentra problemas instalando paquetes MSI confiables, esta opción puede ayudar a que la instalación se complete satisfactoriamente.  Tambien puede intentar deshabilitar Rebajar permisos de grupos Administradores y Usuarios Avanzados.</translation>
     </message>
     <message>
         <location filename="Forms/OptionsWindow.ui" line="410"/>
-        <source>Prompt user for large file migration</source>
-        <translation>Preguntar al usuario sobre migrar archivos grandes</translation>
+        <source>Prompt user for large file migration.</source>
```
Comment (isaak654) [inline SandboxiePlus/SandMan/sandman_es.ts:3189]:
```suggestion
        <source>Prompt user for large file migration</source>
```
See the following remark from the [Localization notes](https://github.com/sandboxie-plus/Sandboxie/discussions/1123):

> ⚠️ You shouldn't ever edit the text in the source tags of the .ts file if you don't apply the same edit on the respective source code file. For example, see the following commit: [3b86441](https://github.com/sandboxie-plus/Sandboxie/commit/3b864411275ac0fa6e60ea1544feb176b470c049).

File: SandboxiePlus/SandMan/sandman_es.ts [hunk 2]
```diff
@@ -3181,13 +3181,13 @@ NO seleccionará: %2</translation>
...
+        <translation>Preguntar al usuario si desea migrar archivos grandes.</translation>
```
Comment (isaak654) [inline SandboxiePlus/SandMan/sandman_es.ts:3190]:
```suggestion
        <translation>Preguntar al usuario si desea migrar archivos grandes</translation>
```

File: SandboxiePlus/SandMan/sandman_es.ts [hunk 3]
```diff
@@ -3181,13 +3181,13 @@ NO seleccionará: %2</translation>
     </message>
     <message>
         <location filename="Forms/OptionsWindow.ui" line="509"/>
-        <source>Note: Msi Installer Exemptions should not be required, but if you encounter issues installing a msi package which you trust, this option may help the installation complete successfully. You can also try disabling drop admin rights.</source>
-        <translation>Nota: Excepciones del instalador MSI no deben ser requeeridas, pero si encuentra problemas instalando paquetes msi que ud. confia, esta opción puede ayudar a que la instalación se complete correctamente.  Ud. puede tambien intentar deshabilitar Solptar Permisos Administrativos.</translation>
+        <source>Note: MSI Installer Exemptions should not be required, but if you encounter issues installing a MSI package which you trust, this option may help the installation complete successfully. You can also try disabling drop admin rights.</source>
```
Comment (isaak654) [inline SandboxiePlus/SandMan/sandman_es.ts:3184]:
```suggestion
        <source>Note: Msi Installer Exemptions should not be required, but if you encounter issues installing a msi package which you trust, this option may help the installation complete successfully. You can also try disabling drop admin rights.</source>
```

See the following remark from the [Localization notes](https://github.com/sandboxie-plus/Sandboxie/discussions/1123):

> ⚠️ You shouldn't ever edit the text in the source tags of the .ts file if you don't apply the same edit on the respective source code file. For example, see the following commit: [3b86441](https://github.com/sandboxie-plus/Sandboxie/commit/3b864411275ac0fa6e60ea1544feb176b470c049).

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: SandboxiePlus/SandMan/sandman_es.ts (hunk 2 above)
Comment (sebadamus) [reply SandboxiePlus/SandMan/sandman_es.ts:3190]: OK

File: SandboxiePlus/SandMan/sandman_es.ts (hunk 3 above)
Comment (sebadamus) [reply SandboxiePlus/SandMan/sandman_es.ts:3184]: OK got it!

**Review by sebadamus — COMMENTED** [review]
There I fixed my working file not to post this changes in future commits.  Hope its ok now.

**Comment by isaak654** [comment]
The file is the same as before, you will need to overwrite it on [this branch](https://github.com/sebadamus/Sandboxie/tree/patch-4/SandboxiePlus/SandMan) in order to see the updated one here.

**Comment by sebadamus** [comment]
> The file is the same as before, you will need to overwrite it on [this branch](https://github.com/sebadamus/Sandboxie/tree/patch-4/SandboxiePlus/SandMan) in order to see the updated one here.

There, have just done it! tell me if it is okay now, sorry for the mess.

https://github.com/sebadamus/Sandboxie/blob/patch-4/SandboxiePlus/SandMan/sandman_es.ts

**Review by isaak654 — APPROVED** [review]

**Pull request merged**